openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx

```diff
@@ -0,0 +1,652 @@
+        return [];
+      }
+    },
+    []
+  );
+
+  // Render condition field based on function selected
+  const getConditionField = (condition: string, name: number) => {
+    const func = filterFunctions?.find((func) => func.name === condition);
+
+    switch (condition) {
+      case 'matchAnyEntityFqn':
+        if (func) {
+          return (
+            <Form.Item className="w-full" name={[name, 'condition']}>
+              <AsyncSelect
+                api={getEntityByFQN}
+                mode="multiple"
+                placeholder="Search by FQN"
+                showArrow={false}
+              />
+            </Form.Item>
+          );
+        }
+
+        break;
+      case 'matchUpdatedBy':
+      case 'matchAnyOwnerName':
+        if (func) {
+          return (
+            <Form.Item className="w-full" name={[name, 'condition']}>
+              <AsyncSelect
+                api={getUsersAndTeamsOptions}
+                mode="multiple"
+                placeholder="Search by Owner"
+              />
+            </Form.Item>
+          );
+        }
+
+        break;
+      default:
+        if (func) {
+          return (
+            <Form.Item className="w-full" name={[name, 'condition']}>
+              <Select
+                mode="multiple"
+                options={
+                  func.paramAdditionalContext?.data?.map((d) => ({
+                    label: startCase(d),
+                    value: d,
+                  })) ?? []
+                }
+                placeholder="Select Event Type"
+              />
+            </Form.Item>
+          );
+        }
+    }
+
+    return <></>;
+  };
+
+  // Watchers
+  const filters = Form.useWatch(['filteringRules'], form);
+  const alertActions = Form.useWatch(['alertActions'], form);
+  const entitySelected = Form.useWatch(['triggerConfig', 'entities'], form);
+  const trigger = Form.useWatch(['triggerConfig', 'type'], form);
+
+  // Run time values needed for conditional rendering
+  const functions = useMemo(() => {
+    if (entityFunctions) {
+      if (!trigger || trigger === AlertTriggerType.AllDataAssets) {
+        return entityFunctions['all'].supportedFunctions.sort();
+      }
+
+      const arrFunctions = entitySelected?.map(
+        (entity) =>
+          entityFunctions[entity as unknown as string].supportedFunctions
+      );
+
+      const functions = arrFunctions
+        ? intersection(...arrFunctions).sort()
+        : [];
+
+      return functions as string[];
+    }
+
+    return [];
+  }, [entitySelected, entityFunctions]);
+
+  const selectedTrigger = useMemo(
+    () => defaultTriggers.find(({ type }) => trigger === type),
+    [defaultTriggers, trigger]
+  );
+
+  // React component which will be septate out next
+  const StyledCard = ({
+    heading,
+    subHeading,
+  }: {
+    heading: string;
+    subHeading: string;
+  }) => {
+    return (
+      <Card bodyStyle={{ background: '#DDE3EA' }} bordered={false}>
```

Comment:
We can move this constant [file](https://github.com/open-metadata/OpenMetadata/blob/5132a18068578648ae465689bfff81ad49cdb99a/openmetadata-ui/src/main/resources/ui/src/constants/constants.ts#L30)